src/foam/dashboard/view/DashboardCitationView.js

```diff
@@ -51,7 +52,7 @@ foam.CLASS({
     },
 
     function openFilteredListView(obj) {
-      var dao = this.__subContext__[obj.listDAOName].where(this.EQ(obj.searchKey, obj.id));
+      var dao = this.__subContext__[obj.data && obj.data.listDAOName || obj.dao].where(this.EQ(obj.searchKey, obj.id));
```

Comment:
@kgrgreer I don't see how listDAOName is setup.  Most CitationViews have a dao property.  Passing 'this' rather than 'this.data', gives openFilteredListView access to the citation's dao property.

Reply:
I think listDAOName is just manually configured in foam.dashboard.view.Table, it isn't calculated in any way.

Reply:
I don't see where openFilteredListView is called, but it's overwritten in a couple spots where they just hard-code their DAO.
https://github.com/search?q=repo%3AnanoPayinc%2FNANOPAY%20openFilteredListView&type=code